orchestra/bots/staffbot.py

```diff
@@ -101,7 +109,13 @@ def staff(self, task_id,
 
         slack_message = self.staffing_success.format(task_id)
         message_experts_slack_group(task.project.slack_group_id, slack_message)
-        return format_slack_message(slack_message)
+        return format_slack_message(
+            command,
+            attachments=[{
+                'color': 'good',
```

Comment:
can we abstract this to a success/error attachement message so we don't have to repeat? or maybe it's not worth it?